analytics/Dockerfile

```diff
@@ -11,15 +11,24 @@ FROM python:3.13-slim AS base
 RUN pip install --no-cache-dir poetry==1.8.2 --upgrade
 
 RUN apt-get update \
+  # Remove existing packages before installing their never versions
+  && apt-get remove --yes \
```

Comment:
This pattern (removing first) was necessary for postgres recently, so now I want to do it everywhere